src/cpp/shared/base.h

```diff
@@ -37,10 +37,29 @@ extern int giMinExtentX;
 extern int giMinExtentY;
 extern int giMaxExtentX;
 extern int giMaxExtentY;
+extern int giWalkingFrom;
```

Comment:
Could you insert some blank lines into this giant block of declarations to make the grouping more apparent?

Reply:
I can but I didn't intend to group them there when I pasted them :D